pylint/lint/expand_modules.py

```diff
@@ -148,13 +146,11 @@ def expand_modules(
                     subfilepath, is_namespace, path=additional_search_path
                 )
                 submodname = ".".join(modpath)
-                result.append(
-                    {
-                        "path": subfilepath,
-                        "name": submodname,
-                        "isarg": False,
-                        "basepath": filepath,
-                        "basename": modname,
-                    }
-                )
+                result[subfilepath] = {
+                    "path": subfilepath,
+                    "name": submodname,
+                    "isarg": False,
```

Comment:
We could have an issue with something that has `isarg` set to `False` clobbering something that has `isarg` set to `True`. Need to think more about that.

Reply:
Addressed with most recent commit on this PR.